rpcs3/Emu/Io/usio_config.h

```diff
@@ -8,18 +8,21 @@ enum class usio_btn
 {
 	test,
 	coin,
+	service,
 	enter,
 	up,
 	down,
-	service,
-	strong_hit_side_left,
-	strong_hit_side_right,
-	strong_hit_center_left,
-	strong_hit_center_right,
-	small_hit_side_left,
-	small_hit_side_right,
-	small_hit_center_left,
-	small_hit_center_right,
+	left,
+	right,
+	taiko_hit_side_left,
+	taiko_hit_side_right,
+	taiko_hit_center_left,
+	taiko_hit_center_right,
+	tekken_button1,
+	tekken_button2,
+	tekken_button3,
+	tekken_button4,
+	tekken_button5,
```

Comment:
if these's such a huge difference between the two, then why didn't you just make two configs ?
Isn't the needed config known before all the translators are called?

Reply:
Pardon, do you mean this?
![Image](https://github.com/RPCS3/rpcs3/assets/17809637/dd8c7cf8-796a-4530-9f75-c647a44fe17d)

Reply:
Touché.
But at the time there wasn't a real separation between games, only a logical one.
Now that you actually change the function based on the game, you might as well add a config for each (if you want to)

Reply:
That's gonna be another work, it's not the aim of this PR though.
Currently it still looks fine visually as I only added 5 and removed 4.